test/tests/selftest/pbs_microsecondlogging.py

```diff
@@ -41,32 +41,33 @@
 
 class TestMicrosecondLogging(TestSelf):
     """
-    Test pbs microsec logging
+    Test log_match() works when start and endtime
+    values are in microseconds
     """
 
-    def setup_microsecondlogging(self, hostname=None, highrestimestamp=1):
+    def switch_microsecondlogging(self, hostname=None, highrestimestamp=1):
         """
         Set micro second logging in pbs.conf
         """
-        du = DshUtils()
+        self.du = DshUtils()
         if hostname is None:
             hostname = self.server.hostname
         a = {'PBS_LOG_HIGHRES_TIMESTAMP': highrestimestamp}
         self.du.set_pbs_config(hostname=hostname, confs=a, append=True)
         PBSInitServices().restart()
         self.assertTrue(self.server.isUp(), 'Failed to restart PBS Daemons')
 
-    def test_microsec_logging(self):
+    def test_log_match_microsec_logging(self):
         """
         Test that log_match will work when microsecond logging
         is turned on and then off
         """
 
         # Turn of microsecond logging and test log_match()
-        self.setup_microsecondlogging(highrestimestamp=1)
+        self.st = time.time()
```

Comment:
Why are you making 'st' and 'et' Class variables? they are local to this test case, so i think you should just declare them as local variables (without the 'self')